extensions/wikia/Email/templates/temporaryPassword.mustache

```diff
@@ -3,8 +3,11 @@
 	<tr>
 		<td class="widest">
 			<p>{{passwordIntro}}</p>
+			{{# username}}
+				<p><pre>{{username}}</pre></p>
```

Comment:
The forgot-password email design didn't specify a preformatted display for the temp password, but @garthwebb added it. Should I remove the ```pre``` tags or leave them? @garthwebb what do you think?